tests/test_model_serializer.py

```diff
@@ -169,7 +170,7 @@ class Meta:
                 file_path_field = FilePathField(path='/tmp/')
         """)
 
-        self.assertEqual(unicode_repr(TestSerializer()), expected)
```

Comment:
Yeah -- though it's a bit odd that `repr` would act differently across platforms...